modules/040-node-manager/hooks/discover_standby_ng.go

```diff
@@ -65,8 +61,22 @@ func standbyNodeGroupFilter(obj *unstructured.Unstructured) (go_hook.FilterResul
 		taints = make([]v1.Taint, 0)
 	}
 
+	needStandby := false
+	maxPerZone := 0
+	if nodeGroup.Spec.NodeType == ngv1.NodeTypeCloudEphemeral {
+		maxPerZone = int(*nodeGroup.Spec.CloudInstances.MaxPerZone)
```

Comment:
> runtime error: invalid memory address or nil pointer dereference

The same is below for MinPerZone.

Reply:
I rely on openapi validation. 
minPerZone/maxPerZone are mandatory for CloudEphemeral NGs.